autogluon/utils/tabular/ml/learner/abstract_learner.py

```diff
@@ -593,3 +534,34 @@ def distill(self, X=None, y=None, X_val=None, y_val=None, time_limits=None, hype
                                                 augmentation_data=augmentation_data, augment_method=augment_method, augment_args=augment_args)
         self.save_trainer(trainer=trainer)
         return distilled_model_names
+
+    @classmethod
+    def load_info(cls, path, reset_paths=True, load_model_if_required=True):
+        load_path = path + cls.learner_info_name
+        try:
+            return load_pkl.load(path=load_path)
+        except Exception as e:
+            if load_model_if_required:
+                learner = cls.load(path_context=path, reset_paths=reset_paths)
+                return learner.get_info()
+            else:
+                raise e
+
+    def save_info(self, include_model_info=False):
+        info = self.get_info(include_model_info=include_model_info)
+
+        save_pkl.save(path=self.path + self.learner_info_name, object=info)
+        save_json.save(path=self.path + self.learner_info_json_name, obj=info)
+        return info
+
+    # TODO: Add data info gathering at beginning of .fit() that is used by all learners to add to get_info output
+    # TODO: Add feature inference / feature engineering info to get_info output
+    def get_info(self, **kwargs):
```

Comment:
should we add call to `get_info` in `fit_summary`?

Reply:
get_info is very heavy in terms of the amount of info it gives, it probably isn't practical to print out. I'm not sure if we want to tie in too many things into fit_summary, since users can just call predictor.info() to get it on its own.